OpenRA.Game/Map/Map.cs

```diff
@@ -206,6 +207,19 @@ static object LoadVideos(MiniYaml y)
 			return videos;
 		}
 
+		[FieldLoader.LoadUsing("LoadMusic")] public ReadOnlyDictionary<string, MiniYaml> Music;
```

Comment:
Not with a `MiniYaml` type (and only with `Dictionary<,>`).
I could add that somewhere.